### PR TITLE
docs: update repo url

### PR DESCRIPTION
**Pull request by Budalebah**
git clone directry -----> https://github.com/eq-lab/wardenprotocol.git

### CODE REVIEW

File: SETUP.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ This page describes the necessary steps to take before running and using Warden
 Clone the repo:
 
 ```bash
-git clone git@github.com:eq-lab/wardenprotocol.git
+git clone https://github.com/eq-lab/wardenprotocol.git & cd wardenprotocol
```
Comment (pitasi) [inline SETUP.md]:
```suggestion
git clone https://github.com/warden-protocol/wardenprotocol.git & cd wardenprotocol
```

**Review by pitasi — CHANGES_REQUESTED** [review]
Hey there, thanks for your contribution!

As things are still moving fast at this stage, most of the docs are now outdated (with the intent of fixing everything by the end of this week).

Can you also update the GitHub org name? I'll merge it afterwards.

Thanks!

File: SETUP.md (hunk 1 above)
Comment (Budalebah) [reply SETUP.md]: can you check now please

Comment (pitasi) [reply SETUP.md]: did you push the changes? I don't see any difference

Comment (Budalebah) [reply SETUP.md]: now yes

**Review by pitasi — APPROVED** [review]

**Pull request merged**